crypto/src/lib.rs

```diff
@@ -718,6 +718,65 @@ impl From<PrivateKey> for PrivateKeySerialized {
     }
 }
 
+ffi::ffi_item! {
+/// Random State of Verifiable Random Function
+#[derive(
+    Debug,
+    PartialEq,
+    Ord,
+    PartialOrd,
+    Eq,
+    Clone,
+    Hash,
+    Decode,
+    Encode,
+    Serialize,
+    Deserialize,
+    IntoSchema,
+)]
+pub struct VRFState {
```

Comment:
let's not put this into `iroha_crypto`. This crate is considered finished and up for audit